easybuild/easyconfigs/d/DCA/DCA++-CMake-CrayGNU-2015.11.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'DCA++'
+version = '1.0'
+
+#consider building everything with eb:
+#preinstallopts: change build_directory
+#disable clean build directory
+
+homepage = 'http://www.itp.phys.ethz.ch/people/schulthess'
+description = """This modules bundles all the dependencies for DCA++, the best DCA ever"""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11'}
+
+dependencies = [
+    ('CMake', '3.3.1'),
```

Comment:
Magma already uses CMake 3.5.0: you could use 3.5.0 also here (if DCA++ is OK with that) or downgrade CMake version on Magma to match 3.3.1 (in order to avoid compiling CMake twice).